src/System.Linq.Expressions/tests/Call/CallTests.cs

```diff
@@ -225,5 +225,56 @@ public static void CallByRefMutableStructIndexWriteBack(bool useInterpreter)
             var m = new Mutable() { X = 41 };
             Assert.Equal(42, lambda(m));
         }
+
+        [Fact]
+        public static void ToStringTest()
+        {
+            // NB: Static methods are inconsistent compared to static members; the declaring type is not included
```

Comment:
Interesting observation made while writing these tests. `DateTime.Now` will render including the type name, but `String.Format(...)` will render without the type name.